src/interfaces/IBlackScholes.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: BUSL 1.1
+pragma solidity 0.8.13;
+
+import "./IVolatilityOracle.sol";
+
+/**
+ * @notice Interface for pricing strategies via Black Scholes method. Volatility
+ * is derived from the Uniswap pool.
+ * Risk free rate can be estimated as Aave's USDC deposit rate (see 
+ * https://linen.app/interest-rates/earn/historical).
+ */
+interface IBlackScholes {
+    function getPutPremium(
```

Comment:
Everything in valorem is a call really. We need to be able to input a valorem option ID here and get back a premium, not so much a uni-v3 pool.